uvicorn/lifespan/on.py

```diff
@@ -9,6 +9,9 @@ def __init__(self, config):
         if not config.loaded:
             config.load()
 
+        if not config.loop_setup:
+            config.setup_event_loop()
```

Comment:
I'm not sure if we need these anymore - may be that we can get away with *only* making the check in the server?